frontend/src/pages/ChainMemberList.tsx

```diff
@@ -412,6 +412,16 @@ function ParticipantsTable(props: {
     );
   }
 
+  function signedUpOn(signedUpToLoopDate: any) {
+    //console.log(signedUpToLoopDate);
+    //let sampleDate = "2023-02-04 15:22:49.682";
+    let day = new Date(signedUpToLoopDate[0]).getDate();
+    let month = new Date(signedUpToLoopDate[0]).getMonth() + 1;
+    let year = new Date(signedUpToLoopDate[0]).getFullYear();
+    let fromattedSignedUpToLoopDate = `${day}/${month}/${year}`;
```

Comment:
Please use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString by the language selected from `{i18n} = useTranslation()`